addons/l10n_uy/models/account_move.py

```diff
@@ -0,0 +1,104 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models, _
+from odoo.exceptions import ValidationError
+
+
+class AccountMove(models.Model):
+
+    _inherit = 'account.move'
+
+    def _get_uruguayan_vat_taxes(self):
+        """ Found the VAT tax groups defined in Uruguay """
+        data_records = self.sudo().env['ir.model.data'].search([
+            ('module', '=', 'account'),
+            ('model', '=', 'account.tax.group'),
+            '|', '|',
+            ('name', 'ilike', 'tax_group_iva_10'),
```

Comment:
Hi @jco-odoo 

At the current moment for this module we needed in order to identify which invoice lines have UY vat taxes. to ensure that we only have one Uruguayan vat tax per line. Also we are checking that not vat taxes are added to Export Invoices documents.

In future  l10n_uy_reports and l10n_uy_edi Odoo modules , we use them to identify and match the taxes with the columns of the Uruguayan VAT book. to generate some TXT and for electronic invoicing to properly enconde the vat taxes on the xml.

We talk about this with @fvz-odoo  @ren-odoo on meeting, I think that we will recommed the user to the the current tax groups not to create new ones

Reply:
Hi @zaoral the scalidraw that @fvz-odoo shared https://app.excalidraw.com/l/4zcUIVVimSO/4yL8A96tUCJ  is actually to cover this use case. By adding the selection field l10n_uy_type, it will allow us to have an easy way to identify the  VAT taxes or any other Tax type and take this as a parameter for future reports or EDI data. 

cc. @jco-odoo

Reply:
Hi @ren-odoo
Yes indeed, I have added the suggested field.
My last comment was meant to be to try answer Jos "(I still need to see why we need it however) What if a user creates a tax group by himself?" 

Now that we have the field we can mark this comment as resolve?
please let me know best regards